### PR TITLE
Run ./autogen.sh instead of autoconf.

### PR DESCRIPTION
**Pull request by IslandCow**
Fixes build for ghostscript.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
For our records, the ghostscript problem is what is causing all of the AFL exceptions.

File: projects/ghostscript/build.sh [hunk 1]
```diff
@@ -52,6 +52,7 @@ make -j$(nproc) libgs
 $CXX $CXXFLAGS $CUPS_LDFLAGS -std=c++11 -I. \
     fuzz/gstoraster_fuzzer.cc \
     -o "$OUT/gstoraster_fuzzer" \
+    -Wl,-rpath=. \
```
Comment (jonathanmetzman) [inline projects/ghostscript/build.sh]: Does this work if I do this:
```
cd /some/path/not/containing/the/fuzzer
/path/to/the/fuzzer
```
?

Comment (jonathanmetzman) [reply projects/ghostscript/build.sh]: I'm not a linking expert, but I think $ORIGIN might be what we need.

Comment (IslandCow) [reply projects/ghostscript/build.sh]: No, I'm pretty sure that won't work.  I've changed it to $ORIGIN and I think it's correct now.

**Comment by IslandCow** [comment]
PTAL.  I've updated the RPATH.  Tested by building in the container, moved contents of /out to /testing.  From /out, ran /testing/gstoraster_fuzzer.  It seems to work.

**Comment by jonathanmetzman** [comment]
I think this worked.

**Pull request merged**